### PR TITLE
[26719] Refactors schedules to removes references to the url

### PR DESCRIPTION
**Pull request by AlexSCorey**
This addresses AAP-26719.  Down stream schedules are broken because the code references a url is accurate upstream, but not accurate downstream.  This pr rips out references to urls in schedules.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "checks": "concurrently --kill-others-on-fail npm:tsc npm:tsc:framework npm:eslint npm:prettier -c auto",
     "upgrade": "npx npm-check-updates --upgrade --target minor --doctor -x '/^cypress/' && npm audit fix || true && npm dedup || true ",
     "cypress": "cypress open",
-    "e2e:awx": "cypress open --browser chrome --e2e --config-file=cypress.awx.config.ts",
+    "e2e:awx": "cypress open --browser firefox --e2e --config-file=cypress.awx.config.ts",
```
[the remark this answers is not in the document]
Comment (AlexSCorey) [reply package.json]: bagh!

File: frontend/awx/views/schedules/SchedulePage/SchedulePage.tsx [hunk 2]
```diff
@@ -152,7 +154,7 @@ export function SchedulePage(props: {
   };
   if (isInventorySource && params.source_id) {
     tabParams.source_id = params.source_id;
-    tabParams.inventory_type = 'inventory';
+    tabParams.inventory_type = INVENTORY_TYPE;
```
Comment (marshmalien) [inline frontend/awx/views/schedules/SchedulePage/SchedulePage.tsx:157]: How are tabParams being used in this component?

**Review by marshmalien — DISMISSED** [review]
Pulled down and tested schedule creation and edit at the top-level 👍

**Review by acruzgon — APPROVED** [review]
lgtm

**Review by acruzgon — APPROVED** [review]

**Pull request merged**